coding-the-legislation/40_legislation_evolutions.md

```diff
@@ -60,67 +61,71 @@ we get the output:
 
 [Read more about how to code parameters](./legislation_parameters.md#parameters-and-time).
 
-## Variable defined only for a specific time intervale
+## Variable defined until a specific date
 
-As the legislation evolves, some fiscal or benefit mechanism appear and disapear. When creating a variable, you can specify the attribute `start_date` and `stop_date` to defines in which time intervale this variable make sense.
+As the legislation evolves, some fiscal or benefit mechanism appear and disapear.  
+For every variable, you can specify the attribute `end`; it defines until when this variable makes sense.
```

Comment:
When reading the whole page, I find it strange to introduce `end` before introducing the concept of dated formulas. I feel it would be much more readable by first introducing the idea that formulas can have validity dates, and then adding the possibility to define a variable-level end date for the last formula. Ending variables happens less often than evolving formulas.